fcrepo-persistence-ocfl/src/main/java/org/fcrepo/persistence/ocfl/impl/DefaultOCFLObjectSessionFactory.java

```diff
@@ -98,7 +99,9 @@ public DefaultOCFLObjectSessionFactory(final File ocflStagingDir, final File ocf
 
     @Override
     public OCFLObjectSession create(final String ocflId, final String persistentStorageSessionId) {
-        final File stagingDirectory = new File(this.ocflStagingDir, persistentStorageSessionId);
+
+        final File stagingDirectory = new File(this.ocflStagingDir,
+                persistentStorageSessionId == null ? "read-only" : persistentStorageSessionId);
```

Comment:
is "read-only" going to be referenced again in the future or is it just for the sake of having a string here? Wondering if it should be a constant.

Reply:
Ah - yeah:  I put that there for the sake of having a string - it won't be referenced elsewhere.